opus/import/populate_obs_mission_voyager.py

```diff
@@ -13,6 +13,11 @@
 import impglobals
 import import_util
 
+VG_TARGET_TO_MISSION_PHASE_MAPPING = {
```

Comment:
Put an underscore in front of the variable name so it stays local to this file

Reply:
done, move the variable to config_data.py because it's used in multiple files (VG_28xx)